src/main/java/nextstep/subway/dto/SectionResponse.java

```diff
@@ -0,0 +1,101 @@
+package nextstep.subway.dto;
+
+import java.util.Objects;
+import nextstep.subway.domain.line.LineStation;
+
+public class SectionResponse {
+
+    private Long id; // 구간 정보 아이디
+    private Long upStationId; // 상행역 아이디
+    private String upStationName; // 상행역 이름
+
+    private Long downStationId; // 하행역 아이디
+    private String downStationName; // 하행역 이름
+
+    private int distance; // 구간 길이
+
+    private Long lineId; // 노선 아이디
+    private String lineName; // 노선 이름
+
+    public static SectionResponse of(LineStation lineStation) {
+        return new SectionResponse(lineStation.getId(),
+                lineStation.getPreStation().getId(),
```

Comment:
```suggestion
                lineStation.getPreStationId(),
```
디미터 법칙을 고려해본다면, 이렇게 해볼수도 있겠네요 :)

Reply:
수정했습니다! 리팩토링할 때 더 꼼꼼히 봐야겠네요 😮